samples/boards/infineon/xmc7200_itcm_dtcm/README.rst

```diff
@@ -0,0 +1,109 @@
+.. _button-sample:
+
+Button
+######
+
+Overview
+********
+
+A simple button demo showcasing the use of GPIO input with interrupts.
+The sample prints a message to the console each time a button is pressed.
```

Comment:
err no, we don't need a board specific sample for a button press when there already is one in-tree